.github/workflows/release.yml

```diff
@@ -0,0 +1,111 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - 'v*'
```

Comment:
What do you think of making sure is using a version "template" instead of getting everything ? Something like - `v*.*.*`

Reply:
IMO `v*.*.*` is an unnecessarily complex optimization.